### PR TITLE
Developemnt

### PR DESCRIPTION
**Pull request by thankgodr**
## This PR contains 

-  Setup for the Todo app
-  Webpack in configuration
-  HtmlWebpackPlugin
- CSS loader and styloader
- Lint code
- Requirements for To Do list: list structure

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const demoData = () => {
+  if (taskManager.taskList.length === 0) {
+    const singleTask = new Todo('Do action one', false, 1);
+    taskManager.addTask(singleTask);
+
+    const singleTask1 = new Todo('Do action two', false, 2);
+    taskManager.addTask(singleTask1);
+
+    const singleTask2 = new Todo('Do action three', true, 0);
+    taskManager.addTask(singleTask2);
```
Comment (ElsonOtake) [inline src/js/index.js]: - I think it's better to create an array of objects (description, completed, and index). It's a project requirement.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @thankgodr,

Good job so far! 🎉 🎉 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/index.js (hunk 1 above)
Comment (thankgodr) [reply src/js/index.js]: @ElsonOtake the object is the taskmanager class. I just use the addtask method to add a single task to the array containing the objects (Task).

**Review by ahmetbozaci — COMMENTED** [review]
### Hello @thankgodr 
### _This is an additional review as a result of the second opinion request._ 




- [ ] [Requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Create%20an%20index.js,index%3A%20%5Bnumber%5D.) are clear please follow the requirements


_
__



### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by t-yanick — APPROVED** [review]
Hi @thankgodr, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**